tests/testInterp.py

```diff
@@ -33,14 +33,8 @@
 import lsst.afw.display.ds9 as ds9
 import lsst.meas.algorithms as algorithms
 import lsst.meas.algorithms.defects as defects
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
-try:
-    type(verbose)
-except NameError:
-    verbose = 0
-    logging.Trace_setVerbosity("algorithms.Interp", verbose)
```

Comment:
I can't find a logger called ``algorithms.Interp`` in the current master.  By running this test with the lowest log level enabled, I see a few debug messages from ``afw.image.Mask`` ("Number of mask planes: 16") but nothing from this package. So I don't know which logger is supposed to be included here.  I included notes about this in the commit message of https://github.com/lsst/meas_algorithms/commit/c33f9c998fe5757b14474aa91ccf447ead665f6f    In the future developers can add meaningful placeholders here when they add logging to the relevant codes.